bigframes/geopandas/geoseries.py

```diff
@@ -0,0 +1,28 @@
+# Copyright 2024 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+from __future__ import annotations
+
+import bigframes_vendored.geopandas.geoseries as vendored_geoseries
+import geopandas.array
```

Comment:
mypy is failing with 

```
bigframes/geopandas/geoseries.py:17: error: Skipping analyzing "geopandas.array": module is installed, but missing library stubs or py.typed marker  [import-untyped]
```

Since there aren't any typing stubs (https://github.com/geopandas/geopandas/issues/3091) best to just ignore this module for now.

```suggestion
import geopandas.array  # type: ignore
```